### PR TITLE
370 mobile crm user guide

### PR DESCRIPTION
**Pull request by digitaldiina**
Main new content in en/mobile/superoffice-mobile/

### Changes in translations

Initially, we'll not translate the Mobile CRM section. The 3 existing pages that were translated in June stay in the source but will not be linked.

* No learn section in *superoffice-mobile/*, always link to en/mobile/superoffice-mobile/index.md as starting point.
* In translations, *learn/\** moved to 1 folder up to *superoffice-mobile/*
* In en:
  * learn/index.md moved 1 up to new main index page for the section
  * learn/scan-business-card.md moved up to company-contact/scan-business-card.md
  * learn/reply-to-request.md] moved 1 up to request/respond.md

### Changes in old EN superoffice-mobile pages

* faq.md inserted at the bottom of security.md (only relates to security and was incorrectly seen as FAQ about the app in general)
* install.md incorporated in getting-started/ to consolidate info

### Images

* New folder with icons in *common/icons/mobile*
* Mobile-related PNGs in *release-notes* compressed with TinyPNG
* CSS change to tag images with "-app-screen" (https://github.com/SuperOfficeDocs/build-docs/pull/93)

### Housekeeping

* docs/en/request/learn/includes/request-owner.md to enable reuse
* docs/en/dashboard/learn/index.md added anchor

### CODE REVIEW

File: docs/en/mobile/superoffice-mobile/getting-started/login.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+The Mobile CRM app is available for all users of SuperOffice CRM 9 or higher. It is distributed via the respective operating system's app store channels.
+
+## Download and install the app
+
+1. Access [Apple AppStore][6] or [Google Play][7] depending on your device.
+
+    ![Mobile CRM download -app-screen][img6]
+
+    You can also open a web browser on your device, go to[online.SuperOffice.com/mobile][3], and click **install** to choose your device type.
```
Comment (SuperOfficeDevNet) [inline docs/en/mobile/superoffice-mobile/getting-started/login.md:25]: Add a space.

```suggestion
    You can also open a web browser on your device, go to [online.SuperOffice.com/mobile][3], and click **install** to choose your device type.
```

File: docs/en/mobile/superoffice-mobile/getting-started/login.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+2. Download and install the **SuperOffice Mobile CRM** app.
+
+3. Start the SuperOffice Mobile CRM app.
+
+4. Choose the connection for your SuperOffice installation.
+
+    ![Mobile CRM: set up -app-screen][img7]
```
Comment (SuperOfficeDevNet) [inline docs/en/mobile/superoffice-mobile/getting-started/login.md:33]: This image is wrong. This we do not do anymore, it is CRM Online only.

File: docs/en/mobile/superoffice-mobile/getting-started/set-up-email.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+author: Bergfrid Dias
+so.date: 07.24.2023
+keywords: mobile, email
+so.topic: howto
+so.version: 10
+language: en
+so.client: mobile
+---
+
+# Setting up an email account
```
Comment (SuperOfficeDevNet) [inline docs/en/mobile/superoffice-mobile/getting-started/set-up-email.md:14]: Setting up email, this is not possible anymore... Ask JP. This is how it looks now:

image

**Review by SuperOfficeDevNet — CHANGES_REQUESTED** [review]
Just a few comments based on feedback from JP.

File: docs/en/mobile/superoffice-mobile/getting-started/set-up-email.md (hunk 3 above)
Comment (digitaldiina) [reply docs/en/mobile/superoffice-mobile/getting-started/set-up-email.md:14]: See commit https://github.com/SuperOfficeDocs/superoffice-docs/pull/803/commits/9c5152f3c277a053fb8ca1a57ef931c0d7daad59

File: docs/en/mobile/superoffice-mobile/getting-started/login.md (hunk 2 above)
Comment (digitaldiina) [reply docs/en/mobile/superoffice-mobile/getting-started/login.md]: See commit https://github.com/SuperOfficeDocs/superoffice-docs/pull/803/commits/f85bc43e9a3f8a2b0f9ef26a80c40deecd6e1619

**Pull request merged**